README.en.md

```diff
@@ -1,16 +1,26 @@
+<p align="center">
+    <b>For ISP that implemented DPI, use <a href="https://github.com/bebasid/bebasit">bebasit</a> to bypass the DPI or turn on IPv6 if the ISP support it.</b>
```

Comment:
`implemented` should be `uses` and for the turn on ipv6, add s at the end of the word `support`